snuba/utils/streams/types.py

```diff
@@ -41,26 +41,3 @@ class Message(Generic[TPayload]):
 
     def get_next_offset(self) -> int:
         return self.offset + 1
```

Comment:
`Message` is also debatably coupled with the `Consumer` (it doesn't get used with the producer, at least as of #665) but this at least seems more generally applicable (in theory) than the consumer errors.